### PR TITLE
Add delete and edit reviews and comments to admin

### PR DESCRIPTION
**Pull request by Durafen**
I have done testing, but more eyes are always welcome.

Closes [Issue #26](https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/26)

### CODE REVIEW

File: www/review [hunk 1]
```diff
@@ -26,6 +25,8 @@ $isOldReview = false;
 // get the game ID
 $id = $_REQUEST['id'];
 
+if ($_REQUEST['userid']) $userid = $_REQUEST['userid'];
+else $userid = $_SESSION['logged_in_as'];
```
Comment (dfabulich) [inline www/review:29]: Wait a minute! This isn't OK. The `$_REQUEST` variable is under the user's control. They can type anything they want in the `&userid` parameter.

For example, in your branch, login as a non-admin user and go here: http://localhost:8080/review?id=ckdsz92q8cws7r7c&userid=1k0mok394bbghxdp

Expected: You're not Jacq, so you shouldn't be allowed to edit her review
Actual: You're not only allowed to see the "edit review" page, you can edit the review, save, and publish it, as the wrong user.

Comment (dfabulich) [reply www/review:29]: I think it should be OK if we do this (but I haven't tested it):

```php
$userid = $_SESSION['logged_in_as'];
if ($_REQUEST['userid'] && check_admin_privileges($db, $userid)) {
    $userid = $_REQUEST['userid'];
}
```

**Review by dfabulich — CHANGES_REQUESTED** [review]
I think I caught a security issue here!

File: www/review (hunk 1 above)
Comment (Durafen) [reply www/review]: Updated the code.
Nice catch, thanks.

**Pull request merged**